Controllers/TaskController.php

```diff
@@ -0,0 +1,114 @@
+<?php
+
+namespace Controllers;
+
+use Models\TaskModel;
+
+class TaskController implements IController
+{
+    private $model;
+    private $fc;
+
+    public function __construct()
+    {
+        $this->model = new TaskModel();
+        $this->fc = FrontController::getInstance();
+    }
+
+    public function createTaskAction()
+    {
+        $imgName = "";
+        if ($_FILES['imageTask']["error"] == UPLOAD_ERR_OK) {
+            $imgName = rand() . $_FILES['imageTask']['name'];
+            move_uploaded_file($_FILES['imageTask']['tmp_name'], 'Media/images/tasks/' . $imgName);
+        }
+        $timeCreated = time();
+        $createdBy = $_POST["created_by"];
+        $executor = $_POST["executor"];
+        $task = $_POST["task"];
+        $status = "новое";
+        $timeStart = time() + (int)$_POST["time_start"] * 3600;
+        $timeEnd = $timeStart + (int)$_POST["time_end"] * 3600;
+        $approvedBy = "Не подтверждена";
+        $res = $this->model->createTask($timeCreated, $createdBy, $executor, $task, $status, $timeStart, $timeEnd, $approvedBy, $imgName);
+        echo $res ? "Задание успешно добавлено!" : "Произошла ошибка. Корректно заполните все поля!";
+        header("Refresh: 3; /view/render/opt/newTask");
+    }
+
+    public function getAllTasksAction()
+    {
+        $_SESSION["allTasks"] = $this->model->getAllTasks();
+        header("Location: /view/render/opt/allTasks");
+    }
+
+    public function deleteTaskAction()
+    {
+        $id = implode(",", $_POST);
+        $this->model->deleteTask($id);
+        header("Location: /task/getAllTasks");
```

Comment:
5. Can we make some abstract function in our parent controller called something like 'redirect' or 'forward'? 
and incapsulate 
`header('Location: ' . $location) `
inside instead of copy pasting it every single time?